pom.xml

```diff
@@ -23,7 +23,7 @@
     <!-- runtime deps versions -->
     <akka.version>2.3.6</akka.version>
     <cassandra-driver-core.version>2.1.0</cassandra-driver-core.version>
-    <chaos.version>0.6.3</chaos.version>
+    <chaos.version>0.6.5</chaos.version>
```

Comment:
you're right. that change comes from https://github.com/doronin/chronos/commit/e55eaa92e8743776f987ec7c70a4d06d23364114 and got here via `git pull --rebase`